src/test/javascript/portal/ui/openlayers/control/SpatialConstraintSpec.js

```diff
@@ -309,22 +308,18 @@ describe('Portal.ui.openlayers.control.SpatialConstraint', function() {
             expect(spatialConstraint._checkSketch(feature)).toEqual(true);
         });
 
-        it('returns false if geometry is too small, shows error', function() {
```

Comment:
The _checkSketch() method previously checked the validity of the geometry *and* called the _showAnteMeridianError(). Now it's only job is to return the validity.

But I now realise I didn't write tests to check my new behaviours so I will do that.